docsource/060_odoo_migration_manager.rst

```diff
@@ -48,3 +48,7 @@ Are migration scripts fired when installing new modules?
    would never ever do so anyway. Developers are free to corrupt the regular
    workings of Odoo if it helps the migration. For instance, from OpenUpgrade
    9.0 on, the workflow engine is disabled during field recomputation.
+
+How are handled auto-installed modules that have been manually uninstalled?
+   Modules with auto_install=True (like IAP) that have been manually uninstalled
+   are automatically installed during the migration.
```

Comment:
We could add a text like "if you dont want to install such module you can use module_change_auto_install in each migration step to force auto_install to False.

https://github.com/OCA/server-tools/tree/18.0/module_change_auto_install